login.py

```diff
@@ -78,14 +40,14 @@ def main():
 
     for s in services:
         if s == "slack":
-            token_path = os.path.join(ns.ramdisk_path, config.slack_token_filename)
+            token_path = config.slack_token_filename
             if ns.slack_channel_id is None:
                 logger.error("The Slack channel ID must be specified")
                 print("Skipping Slack login")
                 continue
```

Comment:
This is not executed when program is started without --slack-channel-id

Reply:
Isn't that intentional? You need both a Slack channel ID and a token to be able to connect to Slack.